packages/tslint-config-loris/index.js

```diff
@@ -0,0 +1,115 @@
+module.exports = {
+    // Add additional rules from https://github.com/buzinas/tslint-eslint-rules
+    'rulesDirectory': 'node_modules/tslint-eslint-rules/dist/rules',
```

Comment:
```diff
- 'rulesDirectory': 'node_modules/tslint-eslint-rules/dist/rules',
+ 'extends': 'tslint-eslint-rules',
```
See https://github.com/buzinas/tslint-eslint-rules#configure-tslint-to-use-tslint-eslint-rules

Reply:
I am wrong. `extends` includes all rules with all default values. Current solution is right.

Reply:
I decided to leave `rulesDirectory` property as safe by default (it will not enable any rules), but specify only name of npm module instead of pull path.